test/ci-operator-integration/expected_files/expected_with_template.json

```diff
@@ -1247,6 +1248,7 @@
               ],
               "image": "${LOCAL_IMAGE_SRC}",
               "name": "test",
+              "resources": {},
```

Comment:
Can we add config to the test so that we inject some resources? :)

Reply:
on it!

Reply:
addressed